### ISSUE TITLE
Make Profiler save data in a readable format

### ISSUE THREAD

**Issue by gerazov** [spyder-ide/spyder#11383]
This is a feature request connected to an old closed issue #1544.

The `Profiler` is really useful, but it would be great to be able to visualise its output. The data can be saved but there is no documentation on its formatting. 

It would be nice to know how to read it, e.g. `pkl` or `csv`, and it would be great if it's in something that allows easy plotting, e.g. a `pandas` dataframe.

**Comment by ccordoba12** [comment]
@gerazov, just to clarify: you want to see the Profiler results in plain text format?

@steff456, please take a look at this one.

**Comment by gerazov** [comment]
Yeah, whatever is easier for you that we can use for making a plot.
